definitions/ext-snowflake_warehouse/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: EXT
+type: SNOWFLAKE_WAREHOUSE
+
+synthesis:
+  identifier: snowflakeVirtualWarehouse
```

Comment:
I just had a look at these events and I don't see any attribute `snowflakeVirtualWarehouse` on it, should it be there @danifitz ?

Reply:
@IreneMLC I fixed capitalisation error, but the `eventType` should be `SnowflakeVirtualWarehouse` and `SnowflakeAccount` - that is the name of the events. I see you added a commit to change it to `eventType` but the following query shows it should be `event_type`

`FROM SnowflakeAccount SELECT event_type`

Reply:
The attribute field that Irene suggested to change as eventType refers to the `SnowflakeAccount` name (e.g. the eventType corresponds to the "table" in NRDB).  The `event_type` field is an attribute of the telemetry.   In the conditions clause what we specify in this case is that we want to match telemetry on the SnowFlakeAccount event space.   Does that make sense?

Reply:
As Galo says your condition was correct (we expect only one condition and in this case it makes sense that it's the `eventType`). The issue is with the `identifier` and `name` fields in both this definition and the other, they should telemetry attributes that allow us to uniquely identify each entity (different for different entities of the same type, like an ID). Sorry for the confusion.